packages/next-admin/src/utils/props.ts

```diff
@@ -71,6 +73,13 @@ export async function getPropsFromParams({
       : undefined,
     message,
     error: searchParams?.error as string,
+    resourcesIdProperty: resources.reduce(
```

Comment:
This is because we need it in the relation cells for the list 

https://github.com/premieroctet/next-admin/pull/81/files#diff-da9757e7a52f98a9e73b240538830ef8f4563116c8674413c074ca37c2253046L92